pyServer/AzureDiskInspectService.py

```diff
@@ -212,47 +228,116 @@ def ParseUrlArguments(self, urlPath, sasKey):
             
         return operationId, mode, modeMajorSkipTo, modeMinorSkipTo, storageAcctName, container_blob_name, storageUrl
 
+    """
+    Validate and parse the given destination Blob Sas Url.
+    """
+    def ValidateAndParseBlobSasUrl(self, blobSasUrl):
```

Comment:
Can you add a comment with a sample (dummy) BlobURI so it is clear how it looks and what the parsing below is doing?